D2547.md

```diff
@@ -1853,11 +1844,39 @@ This more explicit syntax can be used to apply multiple constraints to the retur
 ```cpp
 template<typename R, typename T>
 requires decay_copyable<R> && some_concept<std::decay_t<R>>
-virtual R some_getter(const T& x) = 0;
+R some_getter(const T& x) customisable;
 ```
 
 # Additional Design Discussions
 
+## Syntax
+
+This version of the proposal introduces the `customisable` contextual keyword,
+and uses `customisable`, `default`, `override` and `final` all as trailing keywords
+in function declarations.
+The syntax is of course subject to change.
+
+R0 of the proposal used the `virtual` keyword, on non-member functions,
+as an argument could be made that what we proposes is, in some way,
+a form of static polymorphism.
```

Comment:
Probably also worth adding in a paragraph about why we initially chose the `virtual` keyword.
* Leading keyword harder to miss than trailing one.
* Matches as declaration counterpart to `override` and `final` keywords for member functions, seemed to correspond here.
* Dealing with static polymorphism (already mentioned)
* Needn't impinge on open multi-methods proposal which uses `virtual` on arguments instead of on the function.

Also may want to list other syntaxes considered and debate pros/cons.
e.g. `static virtual`